### PR TITLE
storedMalformedBytecodeHasStartFromTx

### PR DESCRIPTION
**Pull request by hugo-dc**
Tests that a not valid Contract is rejected.

### CODE REVIEW

File: src/GeneralStateTestsFiller/stEWASMTests/validateECIFiller.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: '0'
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/validateECIFiller.yml]: This shouldn't be a contract in the state, but rather a create transaction. Such a contract cannot be deployed.

**Comment by hugo-dc** [comment]
@axic, I updated the test case, binaryen it is deploying the contract, wabn't doesn't deploy it.

**Comment by axic** [comment]
Try using https://github.com/ewasm/hera/pull/436

**Comment by hugo-dc** [comment]
@axic hera seems to be validating that the deployer contract(the one that returns the actual contract) doesn't calls the start function, but doesn't validate the deployed contract.

**Comment by axic** [comment]
> hera seems to be validating that the deployer contract(the one that returns the actual contract) doesn't calls the start function, but doesn't validate the deployed contract.

That's true. The sentinel going to fix that :(

Let's just merge the first part then.

**Comment by axic** [comment]
This needs https://github.com/ewasm/hera/pull/438.

@hugo-dc can you try it with this branch?

**Comment by hugo-dc** [comment]
@axic I'm getting this error:
```
terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<std::system_error> >'                                                                                                                  
  what():  loading /home/hugo/workspace/hera/build/src/libhera.so failed: EVMC ABI version mismatch: Invalid argument
```
I'm using latest aleth (master), and the hera branch you mentioned.

**Comment by axic** [comment]
> I'm using latest aleth (master)

Please do not use aleth master, just keep using the 1.4 release.

**Comment by hugo-dc** [comment]
This is working with the hera branch you posted above.

**Review by axic — APPROVED** [review]

**Review by axic — APPROVED** [review]
Can you rename this test case to reflect the fact that

**Pull request merged**